docs/developer-docs/latest/getting-started/quick-start.md

```diff
@@ -1,150 +1,488 @@
+  }
+
+  ol li {
+    margin-left: 1em;
+    padding-left: .3em;
+  }
+
+  /*
+    We override the :::warning and :::danger callouts for specific uses here.
+    The CSS is scoped so this won't affect the rest of the docs.
+
+    Eventually this will be turned into custom blocks or VuePress components,
+    once I understand better how markdown-it and markdown-it-custom-block work.
+  */
+  .custom-block.congrats,
+  .custom-block.warning,
+  .custom-block.danger {
+    border-left-width: .25rem;
+  }
+
+  .custom-block.warning {
+    background-color: #f8f8f8;
+    border-color: #bbbbba;
+  }
+
+  .custom-block.warning,
+  .custom-block.danger {
+    margin-top: 2em;
+    margin-bottom: 2em;
+
+    .custom-block-title, p, li {
+      color: rgb(44, 62, 80);
+    }
+    a {
+      color: #007eff;
+    }
+  }
+
+  .custom-block.danger {
+    background-color: rgba(129,107,250, .05);
+    border-color: rgb(129,107,250);
+
+    .custom-block-title {
+      color: rgb(129,107,250);
+      font-weight: bold;
+    }
+  }
+
+  .custom-block.details {
+      color: rgb(44, 62, 80);
+  }
+
+</style>
+
+<!-- We use the vuepress-plugin-tabs plugin but customize tabs to look more like buttons -->
+<!-- Not sure why I doesn't work if CSS is scoped 🤷  -->
+<style lang="scss">
+  .el-tabs--card > .el-tabs__header {
+    padding-top: 2em;
+  }
+
+  .el-tabs--card > .el-tabs__header,
+  .el-tabs--card > .el-tabs__header .el-tabs__nav {
+    border: none;
+  }
+
+  .el-tabs--card > .el-tabs__header .el-tabs__item {
+    border-radius: 8px;
+    border: solid 1px rgba(129,107,250,.2);
+    font-size: 125%;
+    height: 60px;
+    line-height: 60px;
+  }
+
+  .el-tabs--card > .el-tabs__header .el-tabs__item:first-child {
+    border-left: solid 1px rgba(129,107,250,.2);
+    margin-right: 20px;
+  }
+
+  .el-tabs--card > .el-tabs__header .el-tabs__item.is-active {
+    background-color: rgba(129,107,250,0.9);
+    color: white !important;
+    font-weight: bold;
+    border-color: rgb(129,107,250);
+  }
+</style>
+
+Strapi offers a lot of flexibility. Whether you want to go fast and quickly see the final result, or would rather dive deeper into the product, we got you covered.
+
+:::warning REQUIREMENTS
+
+* First, make sure [Node.js and npm are properly installed](/developer-docs/latest/setup-deployment-guides/installation/cli.md#step-1-make-sure-requirements-are-met) on your machine.
+* Optionally, you can also [install the Yarn package](https://yarnpkg.com/en/).
 
-<div class="video-container">
-  <iframe width="853" height="480" src="https://www.youtube.com/embed/zd0_S_FPzKg" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-</div>
+:::
 
-_For a step-by-step guide, please take a look at the following steps. This quickstart is really close to the [FoodAdvisor](https://github.com/strapi/foodadvisor) application._
+👇 Let's get started! Using the big buttons below, please choose between:
```

Comment:
Are these buttons' design checked by the design team?

Reply:
I'll have a review with design team as soon as we have a staging server, it's going to be a lot easier :-)

Same for the integration guide buttons at the end of the Hands-on path.